packages/api-server/opentelemetry.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
```

Comment:
can we change file `opentelemetry.js` to `opentelemetry.ts`, we avoid using js in the project.

Reply:
Done